skia/canvas-kit-wasm-bindgen/src/lib.rs

```diff
@@ -44,11 +44,11 @@ pub use typeface::*;
 pub use typeface_factory::*;
 use wasm_bindgen::prelude::*;
 
-pub trait IntoFloat32Array {
-    fn into_float32_array(&self) -> Float32Array;
+pub trait AsFloat32Array {
+    fn as_float32_array(&self) -> Float32Array;
 }
```

Comment:
If we follow Rust naming guidelines, `to_float32_array` seems appropriate.

![image](https://github.com/NamseEnt/namseent/assets/38313680/3f840889-80f7-4587-b5b4-cdaf6f939702)
[Guidelines](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv)

Reply:
Then you should not approve on review dude